packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -263,7 +264,7 @@ class FileCollection extends DocumentCollection {
    *
    * @param  {object} document        A JSON representing a document, with at least a `_type` and `_id` field.
    * @param  {Array}  documents       An array of JSON files having an `_id` field.
-   * @returns 204 No Content
+   * @returns {number} 204 No Content
```

Comment:
Does this mean that doing a "no content" http request would return `204` as a number? Shouldn't it return a http result or maybe void? (I did not use cozy-client enough yet, so this is a real question, I don't know what is the actual behavior)

Reply:
`If a function that is not in externs has no return value, you can omit the @return tag, and the compiler will assume that the function returns undefined.`

according to https://github-wiki-see.page/m/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#return-type-description

Reply:
I will update the MR and remove the @returns tag

Reply:
We added this return because it's a bit special, we had several regression on it, so we added that this method returns a 204 without any content https://github.com/cozy/cozy-client/commit/917ebb576f5d199ba596039285c9f074da79e13b

Reply:
I will not touch to these returns.